src/main/scala/ai/privado/entrypoint/ScanProcessor.scala

```diff
@@ -260,7 +260,15 @@ object ScanProcessor extends CommandProcessor {
     AppCache.init(sourceRepoLocation)
     import io.joern.console.cpgcreation.guessLanguage
     println("Guessing source code language...")
-    val xtocpg = guessLanguage(sourceRepoLocation) match {
+    var languageDetected: Option[String] = None
+    try {
+      languageDetected = guessLanguage(sourceRepoLocation)
+    } catch {
+      case exc: Exception =>
+        println(s"Error Occurred: ${exc.toString}")
```

Comment:
@abhstabs You are printing the error stack trace. I guess you need to show an error message (by calling getMessage() method on exception) instead of the stack trace. And log the entire stack trace in debug log the way we are doing it everywhere else. (We need to make sure the moment we catch any exception, the first job is to log the error in debug mode.)